cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,65 @@
+Note that this authentication method only works for human users, since only humans may have {{ site.data.products.db }} Console identities.
+
+Application users (i.e. service accounts), can authenticate using JWT tokens from your own identity provider. See [Cluster Single Sign-on (SSO) using JSON web tokens (JWT)](../{{site.versions["stable"]}}/sso-sql.html).
+
+Note that this regards SQL access to a specific CockroachDB Cluster, not access to a {{ site.data.products.db }} organization. For the latter, see [Single Sign-On (SSO) for {{ site.data.products.db }} organizations](cloud-org-sso.html).
+{{site.data.alerts.end}}
+
+## Prerequisites
+
+- You must have a user identity on a {{ site.data.products.db }} organization, with access to a cluster or the ability to create a cluster. For help setting up an organization and cluster, see: [Quickstart with CockroachDB](quickstart.html).
+- To authenticate to a specific cluster using SSO, a {{ site.data.products.db }} user must have a corresponding SQL user already [created](../{{site.versions["stable"]}}/create-user.html#create-a-user) on that cluster. {{ site.data.products.db }} generates a SSO SQL username for each console, corresponding to the user's email by the convention `sso_{email_name}`, where `email_name` is everything up to the `@` in an email address, for example the SQL user `sso_docs` would result from `docs@cockroachlabs.com`. `ccloud` will prompt you to make this user if it does not already exist, in which case an admin must create it manually. 
+- [`ccloud`, the {{ site.data.products.db }} CLI](ccloud-get-started.html) must be installed on your local workstation.
+
+## Signing in with Cluster SSO
+
+
+1. First authenticate to your {{ site.data.products.db }} organization. This command will cause your workstation's default browser to open to a {{ site.data.products.db }} authentication portal. Authenticate here as you normally do to the {{ site.data.products.db }} Console. The `ccloud` utility will receive an authentication token from the browser, allowing you to authenticate from the command line.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud auth login --org {your org label}
+	# when running ccloud on a remote host, add the following:
+	# --no-redirect
+	~~~
+
+1. You may then use the `ccloud` utility to authenticate to your {{ site.data.products.db }} cluster, allowing you to access the SQL interface. Your browser will open again as `ccloud` requests a fresh token, although will not need to log in again if you are already logged in.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud cluster sql --sso {your cluster name}
+	# when running ccloud on a remote host, add the following:
+	# --no-redirect
+	~~~
+
+## Troubleshooting
+
+If you get an error that the SQL user is missing, you may need to create it manually. The SQL user's username **must** match the {{ site.data.products.db }} identity's email address, according to the following convention:
```

Comment:
Maybe give what the error actually is, to help Googlers find the info. If you take my suggestion above about adding a section for creating or finding the SQL SSO user, then you could link from here to that section.

Reply:
That would be great for a follow up ticket when I can actually use the functionality.